CHANGELOG.md

```diff
@@ -23,6 +27,15 @@ Mapbox welcomes participation and contributions from everyone.
 #### Known issues
 - If your instrumentation tests use a mocked, prettified JSON response for Mapbox Directions services, you might need to remove white spaces from `"code": "Ok"` substring to `"code":"Ok"`.
```

Comment:
```suggestion
- Precision of the point at which the route is supposed to change behind the location indicator is negatively impacted for long routes. This can present itself by the route vanishing/changing in front the the location indicator.
```

Reply:
@LukasPaczos known issue about while space removal is no longer relevant, right?

Reply:
Right, that's fixed.